src/org/opendatakit/briefcase/ui/MessageStrings.java

```diff
@@ -73,4 +73,15 @@
 
   public static final String ERROR_DIALOG_TITLE =
       "ODK Briefcase Action Failed";
+  
+  public static final String PROXY_HOST = "Host";
+  public static final String PROXY_PORT = "Port";
+  public static final String PROXY_TOGGLE = "Use Proxy";
+  public static final String GENERAL_SETTINGS_STRING =
+		  "<HTML><U>General Settings</U></HTML>";
+  public static final String PROXY_SETTINGS_STRING =
+		  "<HTML><U>Proxy Settings</U></HTML>";
+  public static final String PROXY_SET_ADVICE =
+		  ". If you are behind a proxy, try setting up your proxy details through 'Settings' tab.";
```

Comment:
This will result in two periods if you enter the wrong user name and password. It'll say "Please verify that the URL, your user credentials and your permissions are all correct.."